### ISSUE TITLE
Importing a logger circumvents "logging-f-string (G004)"

### ISSUE THREAD

**Issue by Topher-the-Geek** [astral-sh/ruff#17219]
### Summary

I have G004 active. It works when the LOGGER is setup in the file that commits the sin

```
LOGGER = logging.getLogger("mymodule")

LOGGER.info(f"Hello, {name}")  # correctly flagged
```

But it breaks when the logger is imported

```
from somemodule import LOGGER

LOGGER.info(f"Hello, {name}")  # not flagged
```

Possibly other G rules are broken too. This is the one I noticed. This maybe looks a bit like #12295

**Comment by loRes228** [comment]
https://docs.astral.sh/ruff/settings/#lint_logger-objects

**Comment by Topher-the-Geek** [comment]
Oh, I see. That's going to be a long list, and tedious to maintain. We are adding loggers throughout submodules, but then different modules of that submodule may all import from a shared location.
